### ISSUE TITLE
Zephyr documentation page references 1.5.0, but it's not yet released

### ISSUE THREAD

**Issue by nashif** [zephyriot/zep-jira14#646]
**_Reported by Mark Linkmeyer:_**

See the "Zephyr 1.5.0" link on https://www.zephyrproject.org/doc/.  It shouldn't be there until 1.5.0 is release, right?  Or, if it's intended to explain "what's coming" in the 1.5.0 release, then the link ought to work.

(Imported from Jira ZEP-730)

**Comment by nashif** [comment]
**_by Mark Linkmeyer:_**

_David Kinder_ , assigning this to you since it ought to be fixed quickly to not confuse users.

**Comment by nashif** [comment]
**_by David Kinder:_**

I thought we'd have a 1.5 document set pushed to the server by now, but since we don't, I've submitted a PR to remove the reference: https://gerrit.zephyrproject.org/r/#/c/4228/

**Comment by nashif** [comment]
**_by Mark Linkmeyer:_**

_David Kinder_ , until the PR is processed and the website is fixed, the status should stay in Accepted.  Please move it to Resolved once it's actually fixed, then I will verify it.  Thanks.

**Comment by nashif** [comment]
**_by Mark Linkmeyer:_**

It's not fixed yet.

**Comment by nashif** [comment]
**_by Sharron LIU:_**

the documentation update has specific process to get published (accessible from reader), refer to this linkage, which telling when can we expect the website updated.
https://wiki.zephyrproject.org/view/Website_Documentation

When look into the folder, I found the "dev/" folder should be the one get updated firstly.
https://nexus.zephyrproject.org/content/sites/site/org.zephyrproject.zephyr/

From reader point of view, usually the issue was verified via Zephyr Project website (https://www.zephyrproject.org/doc), that got updated even later.
